samples/excel/89-preview-apis/protect-data-in-worksheet-and-workbook-structure.yaml

```diff
@@ -18,100 +18,145 @@ script:
 
         async function protectDataInWorksheet() {
             await Excel.run(async (context) => {
-                await worksheetHelper(context, true, false);
+                let activeSheet = context.workbook.worksheets.getActiveWorksheet();
+                activeSheet.load("protection/protected");
+
+                await context.sync();
+
+                if (!activeSheet.protection.protected) {
+                    activeSheet.protection.protect();
+                }
+
+                activeSheet.load("name");
+                activeSheet.load("protection/protected");
+
+                await context.sync();
+
+                console.log("Expected: Worksheet '" + activeSheet.name + "' should " + messageHelper(true) + ". Protected? " + activeSheet.protection.protected);
```

Comment:
The user can see visually that most of the buttons on the **Home** ribbon gray out when the sheet is protected and regain their color when the sheet is unprotected. So you don't really need the `console.log` at the end of the method. If you removed it, then you could also get rid of the `context.sync` just above it and the two `load` calls just above that.